tests/data/drogon/ertrun1/share/metadata/fmu_case.yml

```diff
@@ -15,7 +15,6 @@ fmu:
       id: peesv
     description:
       - My case / ERT run description
-    restart_from: x40b09e8-d47f-49b1-8fxx-g52a5119bd37_iter-3
```

Comment:
This should be kept, but testing that we actually still work and that a warning is issued.

Reply:
Are you suggesting that we need to trigger the warning when user load case metadata that contain `restart_from` ? Currently, I only have warning when user set the value.

Reply:
No, this was probably an erroneous comment from me. I see now that these are the test data.